### ISSUE TITLE
[Enhancement] Add Users Trophy Completion Percentage

### ISSUE THREAD

**Issue by akrock** [RePod/psdle#48]
It would be nice to be able to see Trophy completion percentage next to the lists in PSDLE. 

It's hard to gauge what games have actually been played (especially when they exist on multiple platforms with separate trophy lists) without having to go back and forth between another website to see if you have actually played the game before.

As where PSDLE really shines is those of us with 10+ yr old game collections with 100s of games it I think this would be a good addition. As I certainly can't remember all the games I have actually played vs just purchased; and I'm probably not the only one.

Seems like it would be straight forward to scrape the API and merge with the data already available.

Example scraper: 
https://github.com/jimmyhillis/playstation-trophies/blob/master/src/lib/index.js

**Comment by RePod** [comment]
**TL;DR:** Outside of PSDLE's scope, highly likely impossible.

My recommendation is for another service/project to accept a PSDLE import, combine a user's trophy data, and let them export it. It might also be possible for it to run right on top of PSDLE, although that'd be weird.
This may or may not change in the future, but that's just the current situation.
The best I've seen is Catalog responses for PSDLE *may* contain either the total number of trophies or number of individual trophies (cannot recall).

---

While I understand the intent and benefits of adding this feature I also see it as outside of the scope of PSDLE and a bit of feature creep, at least for the foreseeable future. Without naming it, I can think of another store enhancement project that adds review scores of games where reasonable. It's a great addition for those who need it, but useless for those who don't.

Without having verified it myself, I *might* have to fight with [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) to obtain that information in the browser/store context making it near impossible to begin with.
The resource you linked seems to run purely from NPM (not a browser) and aforementioned *review score project* is a pure Chrome extension which can use its extension features to bypass CORS. PSDLE has several versions to reach the most people, and going Chrome exclusive or exclusive features would reduce that parity across the board.
